pkg/ipamd/datastore/data_store.go

```diff
@@ -455,7 +480,7 @@ func (ds *DataStore) ReadBackingStore(isv6Enabled bool) error {
 					}
 					addr := &AddressInfo{Address: ipAddr.String()}
 					cidr.IPAddresses[ipAddr.String()] = addr
-					ds.assignPodIPAddressUnsafe(allocation.IPAMKey, eni, addr)
+					ds.assignPodIPAddressUnsafe(addr, allocation.IPAMKey, allocation.Metadata, time.Unix(0, allocation.AllocationTimestamp))
```

Comment:
Why not convert everything to UTC instead? (i.e.,) time.Unix(0, allocation.AllocationTimestamp).UTC(). I'm not sure if it defaults to UTC. Did we check that? Otherwise might be confusing to have it converted to different time zones.